### PR TITLE
Added Formula for a small shell script wrapper of tmux

### PR DESCRIPTION
**Pull request by hukl**
This is a formula for a rather simple convenience shell script which is a wrapper for tmux to easily create a session with connections to several ssh hosts in separate panes. People who use it asked me for a brew formula this is my attempt of providing one.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
This project is too new and not notable enough for us yet, sorry. Please consider [creating your own tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) instead. Thanks!

File: Library/Formula/hostmux.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Documentation: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Formula-Cookbook.md
+#                http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (mikemcquaid) [inline Library/Formula/hostmux.rb]: Also: please do this in future, thanks.

**Comment by hukl** [comment]
Ok, fair enough - just submitted it because 2-3 people asked me - I thought your answer would be like that and I completely understand :)

**Comment by DomT4** [comment]
Closing per Mike's comments but please do feel free to submit once it passes:
```
brew audit --strict --online
```
Till then creating and looking after the formula in your own tap is a pretty smooth process. Check out the doc Mike highlighted if you're interested :smiley:.

**Pull request closed without merging**